tests/snuba/search/test_backend.py

```diff
@@ -998,3 +1013,53 @@ def test_search_out_of_range(self):
         )
 
         assert set(results) == set([])
+
+    def test_hits_estimate(self):
+        # 400 Groups/Events
```

Comment:
This is pretty slow, but these snuba tests are not the bottleneck for CI (yet) so maybe not so big a deal